terminator_test.go

```diff
@@ -65,6 +76,23 @@ func createTestData(customVersions map[string]string, customTimes map[string]tim
 					HealthStatus:   "Healthy",
 				},
 			},
+			InstanceDetails: integration.InstanceDetails{
```

Comment:
Nice. The gofmt tool should automatically line up the field name to the value for you.